src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -242,4 +264,63 @@ public function getName()
     {
         return 'config';
     }
+
+    /**
+     * Tries to retrieve information about the current Symfony version.
+     *
+     * @return string One of: unknown, dev, stable, eom, eol
+     */
+    private function requestSymfonyState()
+    {
+        $versionInfo = null;
+
+        // get version information from cache or the roadmap
+        $versionCachePath = $this->kernel->getCacheDir().'/version_info.json';
+        if (file_exists($versionCachePath)) {
+            $versionInfo = json_decode(file_get_contents($versionCachePath), true);
+        } else {
+            $versionResponse = @file_get_contents('http://symfony.com/roadmap.json?version='.preg_replace('/-\w+\d*$/', '', $this->data['symfony_version']));
```

Comment:
You should pass a version like `x.y`. A `x.y.z` will be anyway redirected to `x.y`. That would reduce the number of requests to symfony.com.